inlong-tubemq/tubemq-manager/pom.xml

```diff
@@ -30,7 +30,7 @@
     <name>Apache InLong - TubeMQ Manager</name>
 
     <properties>
-        <guava.version>21.0</guava.version>
+        <guava.version>30.0-jre</guava.version>
```

Comment:
There is also a definition for `guava` in `inlong/pom.xml`:

```
        <guava.version>31.0.1-jre</guava.version>
```

It is recommended that only this version be used globally.

Reply:
Will unify later.